suite-common/wallet-constants/src/cardanoConstants.ts

```diff
@@ -1,6 +1,7 @@
 import { BigNumber } from '@trezor/utils';
 
-export const BACKUP_CARDANO_APY = 4.5;
+export const BACKUP_CARDANO_APY = 1.8;
```

Comment:
in the previous release there was 2.4, now it is 1.8, that does not sound right, please do it on backend side and for now, return 2.4